algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -89,6 +89,7 @@ package com.twitter.algebird
  *        (an interval that depends on `eps`) around the truth.
  * @param seed  A seed to initialize the random number generator used to create the pairwise independent
  *        hash functions.
+ * @param
```

Comment:
@DanielleSucher  You're absolutely right! I will add actual comments here as well.